content/docs/1.6.0/references/settings.md

```diff
@@ -814,7 +814,7 @@ See [Kubernetes Cluster Autoscaler Support](../../high-availability/k8s-cluster-
 
 The storage network uses Multus NetworkAttachmentDefinition to segregate the in-cluster data traffic from the default Kubernetes cluster network.
 
-> **Warning:** This setting should change after detaching all Longhorn volumes, as some of the Longhorn system component pods will get recreated to apply the setting. Longhorn will try to block this setting update when there are attached volumes.
+> **Warning:** This setting should change after detaching all Longhorn volumes, as some of the Longhorn system component pods will get recreated to apply the setting. When applying the setting, Longhorn will try to restart all instance-manager, and backing-image-manager pods if all volumes are detached and eventually restart the instance manager pod without instances running on the instance manager.
```

Comment:
```suggestion
> **Warning:** This setting should change after all Longhorn volumes are detached because some pods that run Longhorn system components are recreated to apply the setting. When all volumes are detached, Longhorn attempts to restart all Instance Manager and Backing Image Manager pods.
```

Reply:
> eventually restart the instance manager pod without instances running on the instance manager.

This is unclear. Do you mean that the pod is restarted only when no instances are running on the Instance Manager?

Reply:
Yes, just like what you mentioned.

Reply:
Modified. Thanks.